vendor/vendor.json

```diff
@@ -441,28 +441,28 @@
 			"revisionTime": "2017-04-20T09:04:30Z"
 		},
 		{
-			"checksumSHA1": "vrp0KSG3kZimXujiJMc/NTf2w3M=",
+			"checksumSHA1": "+Yj9sWqspkwTTYsSpnDACT7ZPU0=",
```

Comment:
maybe we should start vendor artefact lib from artefact lib release tags?
now as I look at it I'm not sure which artifact spec version we support after this merge

could be nice to mention this also in api doc (upload supports artifact spec xyz, it's compatible with 1,2,3), looks also like this is a new functionality which may need changelog entry (maybe?)